sdk/textanalytics/Azure.AI.TextAnalytics/perf/Azure.AI.TextAnalytics.Perf/Scenarios/DetectLanguagePerfTest.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Test.Perf;
+
+namespace Azure.AI.TextAnalytics.Perf
+{
+    public sealed class DetectLanguagePerfTest
```

Comment:
It reads better at the command-line if you name the scenarios without the 'Test' suffix.